src/components/experiences/feedback-experiences.js

```diff
@@ -106,15 +106,20 @@ const Actions = styled(DialogActions)`
 const Box = styled(DialogActions)`
   flex-direction: row; 
   align-self: center;  
+  ${window.innerWidth < 960 ? 'margin: 3px' : ''};
+  .gBTTJU {
```

Comment:
what is this class? most likely its auto generated and this wont hold thru every run of the app

Reply:
Yes I believe it was auto generated. I moved the styles in this class one component up and it has the same effect and will work for every run.